gril/grilio.c

```diff
@@ -44,7 +44,7 @@ struct _GRilIO {
 	GIOChannel *channel;			/* comms channel */
 	GRilDisconnectFunc user_disconnect;	/* user disconnect func */
 	gpointer user_disconnect_data;		/* user disconnect data */
-	struct ring_buffer *buf;		/* Current read buffer */
+	struct ring_buffer *ring_buf;		/* Current read buffer */
```

Comment:
Some function below uses the same name, so I renamed it for clarification. Not really important. Removed in #148 .